plugins/scaffolder-backend/src/scaffolder/tasks/NunjucksWorkflowRunner.ts

```diff
@@ -395,7 +400,7 @@ export class NunjucksWorkflowRunner implements WorkflowRunner {
                 task.updateCheckpoint?.({
                   key,
                   status: 'success',
-                  value,
+                  value: value ?? {},
```

Comment:
Given the need for this, what happens if a checkpoint returns 0, false, null, etc.?